federation/pkg/kubefed/init/init_test.go

```diff
@@ -279,6 +292,9 @@ func TestInitFederation(t *testing.T) {
 		if tc.etcdPersistence != "true" {
 			cmd.Flags().Set("etcd-persistent-storage", tc.etcdPersistence)
 		}
+		if tc.imagePullSecrets != "" {
```

Comment:
`imagePullPolicy` is set to a default value, just like `serverImage`. Still need checking?